guide/14-deep-learning/geospatial-deep-learning.ipynb

```diff
@@ -55,16 +55,19 @@
     "\n",
```

Comment:
<p>Add a markdown link to "Install deep learning dependencies..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>